watchdog/main.go

```diff
@@ -202,6 +202,7 @@ func getAdditionalEnvs(config *WatchdogConfig, r *http.Request, method string) [
 		}
 
 		envs = append(envs, fmt.Sprintf("Http_Method=%s", method))
+		envs = append(envs, fmt.Sprintf("Http_ContentLength=%d", r.ContentLength))
```

Comment:
Minor gripe, although I've already used this name. How important should the transport method be to the code dealing with this?

Reply:
What is the gripe, can you explain?

Reply:
Sorry never noticed this. What I was saying was should the variable care and convey that this information is passed via Http?

It doesn't matter anyway, I've added it to the function I wanted to call. Functionality being more important than abstract correctness and all that :wink: :+1:

Reply:
This is how data is passed via CGI to running applications.

https://users.cs.cf.ac.uk/Dave.Marshall/PERL/node200.html

http://www.cgi101.com/book/ch3/text.html

We have just added a namespace to avoid collisions.

Reply:
the point was specifically about the prefix. The linked CGI example just calls it content-length. Like I say it's no big deal at all. It's more of a big-deal for the code I want to include it in because it shouldn't know or care about HTTP. However I could have instead wrapped it in a shell script so I guess it's as much on my use of the ENV as the name.